include/hc.hpp

```diff
@@ -1453,15 +1456,14 @@ accelerator_view::create_marker(memory_scope scope) const {
     std::shared_ptr<Kalmar::KalmarAsyncOp> deps[1]; 
     // If necessary create an explicit dependency on previous command
     // This is necessary for example if copy command is followed by marker - we need the marker to wait for the copy to complete.
-    hc::memory_scope depReleaseScope = hc::no_scope;
-    std::shared_ptr<Kalmar::KalmarAsyncOp> depOp = pQueue->detectStreamDeps(hcCommandMarker, nullptr, &depReleaseScope);
+    std::shared_ptr<Kalmar::KalmarAsyncOp> depOp = pQueue->detectStreamDeps(hcCommandMarker, nullptr);
```

Comment:
Yes.  SDMA copies do not probe the L2 cache so we need to issues a system-scope release before a copy if any kernels have written L2.   The new design does this check before sending the copy.  The previous design allowed release-scope for markers to be "upgraded" to system-release - removing this simplifies the implementation and the create*marker APIs are free from the upgrade baggage and complexity.